### DISCUSSION TITLE
II GS compatibility

### DISCUSSION THREAD

**Discussion by Malaval-Ren** [hanshuebner/AppleII-VGA#15] [General]
I don't find the software compatible to setup it on IIGS mode

Your card is compatible with an IIGS ?
If no an another one is compatible with the IIGS ?

Trying Markadev firmware for the AppleII-VGA https://github.com/markadev/AppleII-VGA/releases/tag/v1.0.0 is it the solution to be able to work on IIGS ?

Thanks

**Comment by hanshuebner** [comment]
I have no IIgs so I can't tell whether the card works with it, sorry.
